servant-docs/src/Servant/Docs/Internal.hs

```diff
@@ -794,6 +794,9 @@ instance HasDocs sublayout => HasDocs (Vault :> sublayout) where
   docsFor Proxy ep =
     docsFor (Proxy :: Proxy sublayout) ep
 
+instance HasDocs sublayout => HasDocs (WithNamedConfig name config sublayout) where
```

Comment:
Doesn't there need to be an instance for `(WithNamedConfig name config s) :> sublayout` too?